raster_api/runtime/src/app.py

```diff
@@ -67,24 +78,36 @@ async def lifespan(app: FastAPI):
 # /mosaic - PgSTAC Mosaic titiler endpoint
 ###############################################################################
 mosaic = MosaicTilerFactory(
-    router_prefix="/mosaic",
+    router_prefix="/searches/{search_id}",
+    path_dependency=SearchIdParams,
     optional_headers=optional_headers,
     environment_dependency=settings.get_gdal_config,
     process_dependency=PostProcessParams,
```

Comment:
Do we still need to include this dependency if we have `tile_dependencies` below?

Reply:
@smohiudd the process dependency is different from the tile_dependency. I see that we have custom post_process method, so if they are still used we should keep the process_dependency